File: algo-method.com/tasks/1062/main.ts
import { readFileSync } from "fs";
let inputLinesIndex = 0;
const inputLines = readFileSync("/dev/stdin", "utf8").split("\n");
const input = () => inputLines[inputLinesIndex++];
const print = (...messages: any) => console.log(...messages);

const yesNo = (b: boolean) => (b ? "Yes" : "No");

const [A, M] = input().split(" ").map(Number);

const ans = (A & M) === M;
print(yesNo(ans));

File: algo-method.com/tasks/1062/main.cpp
#define _GLIBCXX_DEBUG
#include <bits/stdc++.h>
using namespace std;

template <typename T>
void print(T value) {
  cout << value << endl;
}

string yes_no(bool b) { return b ? "Yes" : "No"; }

int main() {
  int A, M;
  cin >> A >> M;

  bool ans = (A & M) == M;
  print(yes_no(ans));
}
